packages/@webex/plugin-meetings/src/member/util.ts

```
@@ -27,384 +27,384 @@
+    (user) => user.id === id || (user.person && user.person.id === id)
+  );
+
+/**
+ * @param {Object} participant - The locus participant object.
+ * @param {Boolean} isGuest
+ * @param {String} status
+ * @returns {Boolean}
+ */
+MemberUtil.isNotAdmitted = (participant: any, isGuest: boolean, status: string): boolean =>
+  participant &&
+  participant.guest &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)